### PR TITLE
Add support for django 1.8 and 1.9

### PR DESCRIPTION
**Pull request by LilyFirefly**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 SHELL := '/bin/bash'
 
 test:
-	python form_models/tests/run.py
+	coverage run form_models/tests/run.py
+	coverage report
```
Comment (KevinEtienne) [inline Makefile]: `show-missing`?

**Comment by Ian-Foote** [comment]
@incuna/backend Review?

File: Makefile (hunk 1 above)
Comment (adam-thomas) [inline Makefile]: And the option that ignores 100% coverage lines?  I can't remember what it's called. :)

**Comment by adam-thomas** [comment]
LGTM other than nitpicking on `coverage report` above :)

File: Makefile (hunk 1 above)
Comment (Ian-Foote) [inline Makefile]: It's not really needed - there's only 5 files covered.

Comment (Minglee01) [inline Makefile]: is there a chance that more files will be added?

Comment (Ian-Foote) [inline Makefile]: Very low - this project is basically in maintenance mode.

**Comment by adam-thomas** [comment]
:+1:

**Pull request merged**